### DISCUSSION TITLE
Setup EXE für Jarvis-Windows

### DISCUSSION THREAD

**Discussion by djalexz** [Zefau/ioBroker.jarvis#2242] [Show and tell]
Hi @Zefau,

ich wollte jetzt auch mal bisschen was zurückgeben für die ganze Hilfe, die ich hier immer bekomme.
Habe ein Setup für die Windows App Variante von Jarvis erstellt.

v3.1.3-beta.29 

So ist es eine ganz einfache Installation, die auch von weniger versierten Benutzern genutzt werden kann.

Hoffe, dass es euch hilft.

https://filehorst.de/d/eigJxneb

PS: Sollte es hier die falsche Kategorie sein, dann bitte verschieben.

LG Alex

**Comment by Zefau** [comment]
Aktuell ist es ja eigentlich nur das Entpacken eines Zip-Archivs? Ist es so viel einfacher via einer Install-EXE? Wie hast du das gemacht? Ggf. kann ich das auch von "Haus aus" anbieten.

**Comment by djalexz** [comment]
Ja das stimmt schon.
Ggfls möchte es aber der User als Verknüpfung auf dem Desktop oder im Startmenü oder so. Wie ein ganz normales installierbares Programm halt.
War nur so ne Idee...
Mit inno Setup https://jrsoftware.org/isdl.php
Kann dir das Script dafür morgen gerne hochladen.

**Reply by djalexz** [reply]
Moin,

hier der Code zum Kompilieren des Setups.
Natürlich müssen die Pfade noch angepasst werden:

[Setup.txt](https://github.com/Zefau/ioBroker.jarvis/files/10821229/Setup.txt)

Die Datei als .iss abspeichern und mit Inno Setup öffnen.
